### DISCUSSION TITLE
Avoiding Linux bridge creation

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1549] [Ideas]
Could https://github.com/ipspace/netlab/blob/e5f6268db24f70a5d0abaa5564ee5fa30a21068e/netsim/augment/links.py#L869 be changed to
```
if link.type in (['p2p','vlan_member'] + VIRTUAL_INTERFACE_TYPES):
```

**Comment by ipspace** [comment]
I'm guessing that if you stop using **link.type** for all sorts of unrelated things and leave the **p2p** value to mean "point-to-point link", you won't have a problem any more.
